scripts/check-chains.ts

```diff
@@ -0,0 +1,62 @@
+import fs from 'node:fs';
```

Comment:
This was formerly called `validate-chains`. "Validate" refers to RPC calls to contracts in this repo, and "check" is better aligned with the lingo here.